contrib/pom.xml

```diff
@@ -604,10 +604,10 @@
       <optional>true</optional>
     </dependency>
     <dependency>
-      <!-- Kinesis depends on HttpPatch which requires httpclient 4.3.5 -->
+      <!-- Kinesis depends on HttpPatch which requires httpclient 4.3.6 -->
```

Comment:
What Kinesis actually depends on? Why does it need to be documented here instead of a JIRA or a commit?